docs/spec/channel.md

```diff
@@ -189,6 +193,15 @@ Channels MUST attach a bearer token to all outgoing requests, likely in the form
 of a JWT. This bearer token MUST use an identity associated with the Channel,
 not the individual Subscription.
 
+##### Retries
+
+Channels SHOULD retry resending CloudEvents when they fail to either connect or send CloudEvents to subscribers.
+
+Channels SHOULD support various retry configuration parameters, including, but not limiting to:
```

Comment:
```suggestion
Channels SHOULD support various retry configuration parameters, including, but not limited to:
```

Reply:
fixed. (didn't see the commit suggestion button. Next time!)

Reply:
Probably for the best, I think the CLA bot doesn't like it when you accept the suggestions (as they are attributed to the suggester).